src/LiveEditor.js

```diff
@@ -46,28 +52,33 @@ const Err = styled(LiveError)([], {
 
 export default ({
   code,
-  scope,
-  render
-}) => (
-  <Box mb={4}>
-    <ScopeConsumer defaultScope={scope}>
-      {scope => (
-        <LiveProvider
-          code={code}
-          scope={scope}
-          mountStylesheet={false}
-          transformCode={transformCode}>
-          {typeof render === 'function' ? (
-            render({ code, scope })
-          ) : (
-            <React.Fragment>
-              <Preview />
-              <Editor />
-              <Err />
-            </React.Fragment>
-          )}
-        </LiveProvider>
-      )}
-    </ScopeConsumer>
-  </Box>
-)
+  scope = {},
+  render,
+  mdx
```

Comment:
I was thinking of having a `Lang` prop, though I’m not sure what other languages might work here... maybe tsx in the future?

Reply:
Yeah, would prolly be wise. That was how I originally added MDX handling but then made it MDX specific. Will update that this afternoon. 👍